hare/messagevalidation.go

```diff
@@ -49,25 +49,7 @@ func (ev *eligibilityValidator) validateRole(m *pb.HareMessage) bool {
 }
 
 // Validates eligibility and signature of the provided message
-func (ev *eligibilityValidator) Validate(m *pb.HareMessage) bool {
-	data, err := proto.Marshal(m.Message)
-	if err != nil {
-		ev.Error("Validate message failed: failed marshaling inner message")
-		return false
-	}
-
-	// verify signature
-	verifier, err := NewVerifier(m.PubKey)
-	if err != nil {
-		ev.Warning("Validate message failed: Could not construct verifier ", err)
-		return false
-	}
-	res, _ := verifier.Verify(data, m.InnerSig)
-	if !res {
-		ev.Warning("Validate message failed: invalid message signature detected ")
-		return false
-	}
-
+func (ev *eligibilityValidator) Validate(m *Msg) bool {
```

Comment:
In the deleted code there you validated the signature. Where do you do this validation in the new code?